File: javascript/6kyu_Bouncing Balls.js
function bouncingBall(h,  bounce,  window) {

    if (h <= 0 || (bounce <= 0 || bounce >= 1) || window >= h) {
        return -1;
    } else {
        return Math.floor(Math.log(window/h)/Math.log(bounce)) * 2 + 1;
    }

}

File: Codewars/Python/6kyu/6kyu_ Bouncing Balls.py
def bouncingBall(h, bounce, window):
    import math
    if h <= 0 or bounce <= 0 or bounce >= 1 or window >= h:
        return -1
    else:
        return math.floor(math.log10(window/h)/math.log10(bounce)) * 2 + 1